Makefile

```diff
@@ -149,7 +149,7 @@ vet: ## Run go vet against code.
 
 .PHONY: test
 test: generate gotestsum ## Run tests.
-	$(GOTESTSUM) --junitfile $(ARTIFACTS)/junit.xml -- $(GO_TEST_FLAGS) $(shell $(GO_CMD) list ./... | grep -v '/test/') -coverprofile $(ARTIFACTS)/cover.out
+	$(GOTESTSUM) --junitfile $(ARTIFACTS)/junit.xml -- $(GO_TEST_FLAGS) $(shell $(GO_CMD) list ./... | grep -v '/test/') -coverpkg=./... -coverprofile $(ARTIFACTS)/cover.out
```

Comment:
what is this change for?

Reply:
since we are using reconciler in integration tests we need to enable coverage of this external package. This option helps calculate coverage for reconciler package